app/src/main/res/values/strings.xml

```diff
@@ -325,5 +326,7 @@
     <string name="archive_create_new_archive_title">Create New Archive</string>
     <string name="archive_archive_name_hint">Archive Name</string>
     <string name="archive_archive_type_hint">Archive Type</string>
-    <string name="archive_delete_archive_title">Are you sure you want to permanently delete this archive?</string>
+    <string name="dialog_delete_archive_title">Are you sure you want to permanently delete this archive?</string>
+    <string name="dialog_switch_archive_title">Switch to The %s Archive?</string>
+    <string name="dialog_switch_archive_text">In order to access this content you need to switch to The %s Archive</string>
```

Comment:
If a user would name their Archive "The Wedding" this will look weird with 2 The

Reply:
lets keep it like this if this is how its used everywhere